src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Hooks.java

```diff
@@ -38,27 +31,40 @@
 
     private static File dir;
     private static EventFiringWebDriver driver;
+    private static Properties config = new Properties();
     private static final int DELAY = 30;
     private static String screenshotPath;
+    private static FileInputStream fis;
     private static String downloadPath = System.getProperty("downloadPath");
-
+    private static String browser;
     /**
      * Launch ChromeDriver.
+     * @throws IOException
      */
     @Before("~@RegressionAPI")
-    public void openBrowser() {
-        /* To run chrome with the developer tools window automatically opened re-able these lines.
-         *
-        ChromeOptions chromeOptions = new ChromeOptions();
-        chromeOptions.addArguments("--auto-open-devtools-for-tabs");
-        driver = new ChromeDriver(chromeOptions);
-        driver.manage().window().maximize(); // does not work on CI machine, sometimes useful locally
-         */
+    public void openBrowser() throws IOException {
+
+        if (System.getenv("browser") != null && !System.getenv("browser").isEmpty()) {
+            browser = System.getenv("browser");
+            if (browser.equalsIgnoreCase("firefox")) {
+                System.setProperty("webdriver.gecko.driver", System.getProperty("PathtoGeckodriver") + File.separator + "geckodriver");
+            }
+        } else {
+            fis = new FileInputStream(
+                    System.getProperty("user.dir") + File.separator + "src"
+                            + File.separator + "test" + File.separator + "resources"
+                            + File.separator + "Config.properties");
+            config.load(fis);
+            browser = config.getProperty("browser");
+            if (browser.equalsIgnoreCase("firefox")) {
+                System.setProperty("webdriver.gecko.driver", System.getProperty("PathtoGeckodriver") + File.separator + "geckodriver.exe");
```

Comment:
If we have to get the path to the gecko driver from the 'getproperty' call anyway, why not just put the full driver in that value, and include the actual file name? The way it's currently set up, we're assuming someone is running locally on Windows, but there's no reason that needs to be the case. Could move this entire statement outside of the if/else thing and leave it with 
```suggestion
                System.setProperty("webdriver.gecko.driver", System.getProperty("PathtoGeckodriver"));
```

Reply:
Also, no reason not to set the property even if we're not using firefox; it won't hurt to set it (unless it fails if null), but it simplifies things if we do it automatically

Reply:
so path to gecko driver is only needed for jenkins so i have updated hooks.java now